### ISSUE TITLE
Combining frozen columns, autoHeight with a header row resizes the grid too small

### ISSUE THREAD

**Issue by fheck** [6pac/SlickGrid#521]
When having a grid with those 3 properties set, the last row of the grid (and more importantly the horizontal scroll bar) is cur of by the height of the header row. 
In particular I'm using Angular-Slickgrid in the version 2.19.0 which pulls SlickGrid an dependency in version 2.4.25. 

As discussed [here](https://github.com/ghiscoding/Angular-Slickgrid/pull/529#issuecomment-660184724) I've looked into issue #342 but I looks like the fix is not enough for the case where frozen columns are set.

I will create a PR with changes that seem to fix the problem for me, but I can't tell if that has any untended consequences elsewhere.
In short, it looks to me as if the value of `headerRowH` (and the other two values) [here](https://github.com/6pac/SlickGrid/blob/9d453935cebbe366e842a7a03a46587b03fa9d27/slick.grid.js#L3440) is not calculated in the case of `autoHeight` but its height is missing [here](https://github.com/6pac/SlickGrid/blob/9d453935cebbe366e842a7a03a46587b03fa9d27/slick.grid.js#L3486). Moving this calculations up into its own if that is also executed when frozen columns are enabled
